### PR TITLE
[Docs] Convert remaining code snippets in docs

### PR DESCRIPTION
**Pull request by tlrx**
This commit converts the last remaining unconverted code snippets in docs.

### CODE REVIEW

File: docs/build.gradle [hunk 1]
```diff
@@ -19,30 +19,6 @@
 
 apply plugin: 'elasticsearch.docs-test'
 
-/* List of files that have snippets that probably should be converted to
- * `// CONSOLE` and `// TESTRESPONSE` but have yet to be converted. Try and
- * only remove entries from this list. When it is empty we'll remove it
- * entirely and have a party! There will be cake and everything.... */
```
Comment (tlrx) [inline docs/build.gradle]: :cake: :tada:

**Review by rjernst — COMMENTED** [review]
The scripting parts LGTM

File: docs/build.gradle (hunk 1 above)
Comment (nik9000) [reply docs/build.gradle]: 🍰 🎉

**Review by nik9000 — APPROVED** [review]
❤️

**Comment by tlrx** [comment]
Thanks @nik9000 @rjernst

File: docs/build.gradle (hunk 1 above)
Comment (archanid) [reply docs/build.gradle]: :cake: 🎉

**Pull request merged**